src/libcollections/slice.rs

```diff
@@ -343,15 +343,22 @@ impl<T> [T] {
         core_slice::SliceExt::last_mut(self)
     }
 
-    /// Returns the element of a slice at the given index, or `None` if the
-    /// index is out of bounds.
+    /// Returns a reference to an element or subslice depending on the type of
+    /// index.
+    ///
+    /// - If given a position, returns a reference to the element at that
+    ///   position or `None` if out of bounds.
```

Comment:
Not to mention that every reference to `None` seems to be different. There's (all in non-std crates that get re-exported to std)
```
[`None`]: ../../std/option/enum.Option.html#variant.None
[`None`]: option/enum.Option.html#variant.None
[option]: ../../std/option/enum.Option.html
```
Which is the preferred one?